app/containers/MassEnergizeSuperAdmin/Events/notifcation-settings/SavedNudgeSettings.js

```diff
@@ -2,22 +2,43 @@ import React, { useState } from "react";
 import { OPTIONS } from "./EventNotificationSettings";
 import { objArrayToString, pluralize, smartString } from "../../../../utils/common";
 
-const SavedNudgeSettings = ({ profiles, editSettings, removeProfile }) => {
+const SavedNudgeSettings = ({ profiles, editSettings, removeProfile, getStarted }) => {
   const common = {
     marginRight: 10,
     textDecoration: "underline",
     fontWeight: "bold",
     color: "#428BCF"
   };
 
-  if (!profiles?.length) return <p style={{ padding: 10, color: "black" }}>No saved settings yet... The settings you make for any group of communities will be available here.</p>;
+  if (!profiles?.length)
```

Comment:
Pls wrap the if block in curly braces going forward, especially for multi-line blocks like this.
Otherwise some interpreters will have a hard time telling where it ends, what is part of the block and what's not.
Indentation doesn't help here, cos white spaces are ignored in the parsing process;
The only reason this is not causing the error is because of the parenthesis but in general pls avoid writing if statement bodies without braces to prevent unintended consequences, except one-liners at the end.